webapp/channels/build/emoji/make_emojis.mjs

```diff
@@ -389,14 +379,6 @@ export const EmojiIndicesByUnicode = new Map(${JSON.stringify(emojiIndicesByUnic
 
 export const CategoryNames = ${JSON.stringify(categoryNames)};
 
-export const CategoryMessage = new Map(${JSON.stringify(Array.from(categoryDefaultTranslation))});
-
-export const CategoryTranslations = new Map([${categoryTranslations}]);
-
-export const SkinTranslations = new Map([${skinTranslations.join(', ')}]);
-
-export const ComponentCategory = 'Component';
```

Comment:
There's a bunch of stuff that was generated and put into the `emoji.ts` that didn't really need to be. While we get the names of the emoji categories from `emoji-data-source` here, I don't think we need to store all of the labeling information here because we don't get that from the library. The same thing goes for skin tones as well.

I'm not sure what `ComponentCategory` was previously used for, but the value isn't used currently